tox.ini

```diff
@@ -16,11 +16,11 @@ commands = {posargs:coverage run runtests.py}
 # Configure which test environments are run for each Github Actions Python version.
 [gh-actions]
 python =
-    3.6: py36-django{22,32}
-    3.7: py37-django{22,32}
-    3.8: py38-django{22,32,40}
-    3.9: py39-django{22,32,40}
-    3.10: py310-django{22,32,40}
+    3.6: py36-django{22,32},functional
```

Comment:
I'm not familiar with the function tox keyword. Quick google search didn't yield results.

Can you explain it's purpose?

Reply:
Oh scratch that, its the test environment called functional.